cumulus/parachains/runtimes/collectives/collectives-westend/src/ambassador/tracks.rs

```diff
@@ -246,9 +249,9 @@ impl pallet_referenda::TracksInfo<Balance, BlockNumber> for TracksInfo {
 						ceil: Perbill::from_percent(50),
 					},
 				},
-			),
+			},
 		];
-		&DATA[..]
+		DATA.iter().map(Cow::Borrowed)
 	}
```

Comment:
we could also implement `info` method manually to do the binary search.

Reply:
We should have created a `static` parameter just to store the `DATA` array outside `TracksInfo::tracks`. If there's a way to achieve something similar without compromising the current performance, could be good in the future.